src/main/scala/nl.knaw.dans.easy.springfield/Command.scala

```diff
@@ -213,6 +221,20 @@ object Command extends App
     } yield collections
   }
 
+  private def getPresentationList(user: String): Try[Seq[(String,String)]] = {
+    for {
+      xml <- getXmlFromPath(Paths.get("user", user, "collection"))
+      presentations <- Try { listPresentations(xml) }
+    } yield presentations
+  }
+
+  private def getFileList(user: String, presentationId: String): Try[Seq[(Int, String,String, Boolean)]] = {
+    for {
+      xml <- getXmlFromPath(Paths.get("user", user))
+      files <- Try { listFiles(xml, presentationId.toInt) }
```

Comment:
see other comment about this commandline option. it would be easier if it would accept the same input as `list-presentations` outputs, so not the `referid` but the human readable`@id`. i think you would need the `presentation` in the `collection` and not the top-level `presentation`.